pageserver/src/bin/pageserver.rs

```diff
@@ -276,7 +275,17 @@ fn start_pageserver(
     let pageserver_listener = tcp_listener::bind(pg_addr)?;
 
     // Launch broker client
-    WALRECEIVER_RUNTIME.block_on(pageserver::broker_client::init_broker_client(conf))?;
+    let broker_client = WALRECEIVER_RUNTIME
+        .block_on(async {
+            // Note: we do not attempt connecting here (but validate endpoints sanity).
```

Comment:
Perhaps the method should had been called connect_lazy, but verified this comment does match reality.

Reply:
Yeah, had the same thought but let's have it in a different PR